LocalPackages/AIChat/Sources/AIChat/UserText.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  UserText.swift
+//  DuckDuckGo
+//
+//  Copyright © 2024 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+
+public struct UserText {
+    public static let aiChatTitle = NSLocalizedString("aichat.title", value: "DuckDuckGo AI Chat", comment: "Title for DuckDuckGo AI Chat. Should not be translated")
```

Comment:
We also have `NotLocalizedString` in BSK which could be used here to avoid the string getting accidentally translated, but tbh I've never used it and not sure if it just swaps in seamlessly. 😄

Reply:
Yep, I was using it but I saw that AI Chat is in our translation memory to no translated it. So decided against copying the NotLocalizedString to the package for now.